deploy/charts/harvester/templates/harvester-storageclass.yaml

```diff
@@ -0,0 +1,19 @@
+{{ if .Values.longhorn.enabled -}}
```

Comment:
Should we put this `storageclass` into `harvester-installer`, as there has assumed that the `storageclass` is longhorn.

And the `rancher-monitoring` in `harvester-installer`, has already claimed PVC from `longhorn`.

Change `default storageclass` may also cause break with previous version.

Reply:
Another benefit:

When this `harvester-longhorn` storageclass is in `harvester-installer`, it's convenient to change and make ISO, e.g., set replica to `1` instead of `3`.

Reply:
From discussion with @guangbochen, If the user needs a different replica setting. He can create a new storage class on day2 and make it as default if needed.

Reply:
We will keep the original `longhorn` and adding a new storage class like `harvester-longhorn`(the default one), the exisitng PVC will remain unchanged.

Reply:
I would suggest to put this new one, in `harvester-installer`, in `rancherd-10-harvester`, it is defined before `managedchart` reference it.

Reply:
It's better to stay in the helm chart here, considering the upgrade path. rancherd-10-harvester doesn't run in upgrade path.